dipy/tracking/fbcmeasures.pyx

```diff
@@ -0,0 +1,334 @@
+import numpy as np
+cimport numpy as cnp
+cimport cython
+
+cimport safe_openmp as openmp
+from safe_openmp cimport have_openmp
+from cython.parallel import parallel, prange, threadid
+
+from scipy.spatial import KDTree
+from scipy.interpolate import interp1d
+from math import sqrt, log
+
+from dipy.data import get_sphere
+from dipy.denoise.enhancement_kernel import EnhancementKernel
+from dipy.core.ndindex import ndindex
+
+cdef class FBCMeasures:
+
+    cdef int [:] streamline_length
+    cdef double [:, :, :] streamline_points
+    cdef double [:, :] streamlines_lfbc
+    cdef double [:] streamlines_rfbc
```

Comment:
I believe that the class variables need to be defined this way in Cython since they require an extension type. Check out this [Cython docpage](http://docs.cython.org/src/userguide/extension_types.html)